### DISCUSSION TITLE
车载应用平台上的交规小助手（驾考小助手）

### DISCUSSION THREAD

**Discussion by lindsey-chang** [InternLM/Tutorial#487] [project]
# 车载应用平台上的交规小助手（驾考小助手）

## 研究目的
随着新能源汽车越来越火热，各大车企厂商也开始注重车载平台应用的开发，本项目旨在开发一个车载平台上的小应用：**交规小助手**，也可作为有意向考取驾驶证同学进行题目问答解答的**驾考小助手**

## 研究方法

科目一、科目四的题库可以便捷的改为SFT的训练数据集格式
外挂文本形式的数据库进行RAG

## 数据集

科目一、科目四的题库

## 期待你的加入
希望你
- 🥹 掌握基本的Python用法，能用Python进行开发
- 😇 完成书生浦语训练营的所有基础作业
- 🥳 热爱知识，热爱学习，不拖延
- 🥰 一起努力，一起学习

**Comment by lindsey-chang** [comment]
有意向加入的同学们请联系我的微信：LindseyChang20

**Reply by lindsey-chang** [reply]
有意向加入的同学请附上自我介绍喵

**Comment by Doublefei-Giser** [comment]
dd

**Comment by lindsey-chang** [comment]
自我介绍：我目前是就读于University of Wollongong的硕士研究生，计算机专业，选修过多门机器学习方向的课程，有基本的能进行机器学习项目实战的基础，之前经常在Colab上进行课程项目的训练，会使用Latex进行报告的撰写。之前也使用[`llama-factory`](https://github.com/hiyouga/LLaMA-Factory)进行过基于ChatGLM的DPO微调的流程，有一定的大模型基础知识，希望能和大家一起努力

**Comment by lindsey-chang** [comment]
已满：
Lindsey（24班）、田帅（10班）、ljqiff（2班）、tackhwa（18班)、黑桃（18班）
